config/initializers/0_clowder-config.rb

```diff
@@ -40,10 +40,10 @@ def build_endpoint_url(endpoint, config)
     if kafka_security_protocol == 'sasl'
       cacert = first_kafka_server_config&.dig('cacert')
       if cacert.present?
-        kafka_server_config[:ssl_ca_location] = 'tmp/kafka_ca.crt'
```

Comment:
Searching for a file by a known pattern has similar weakness implications. If we need to use a single file, then it can be on predefined location with right permissions, however I'd recommend here to check the content of the file as well to be sure.